### ISSUE TITLE
Another false positive of Style/FormatStringToken

### ISSUE THREAD

**Issue by iGEL** [rubocop-hq/rubocop#6308]
I've found an edge case, where the fixes for #5290 and #5398 don't work. I would be fine to just close this if you tell me, that this is too unlikely to hit to justify a fix, but I wanted to bring it to your attention at least.

--------

## Expected behavior

Rubocop should not report an offense for this code:

```ruby
def format(date)
  date.strftime("%d/%m/%Y")
end
```

## Actual behavior

It reports `C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over unannotated tokens (like %s).` This does not happen if I rename the method to something else than `format`!

## RuboCop version

```
0.58.2 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-linux)
```

**Comment by Drenmi** [comment]
Thanks for the report! I can not reproduce this issue on latest `master`. 🤔 Can you try it on 0.59.1?
